src/app/app.html

```diff
@@ -1,8 +1,13 @@
 <ion-split-pane>
   <ion-menu [content]="content">
-    <ion-header aria-hidden="true">
+    <ion-header>
       <ion-toolbar color="primary">
-        <ion-title>PVTrAck</ion-title>
+        <ion-title aria-label="Peeveetrack">PVTrAck</ion-title>
+        <ion-buttons end>
+        <button ion-button menuClose aria-label="Close the menu" tabindex=2 *ngIf="this.platform.is('core') == false">
```

Comment:
See my comment on how to fix this design-pattern wise, but I think the `this` keyword is why this isn't working